Project/src/main/resources/templates/editor.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <title>Dashboard</title>
-    <script src="webjars/jquery/1.9.1/jquery.min.js"></script>
+    <script src="webjars/jquery/1.9.1/jquery.js"></script>
```

Comment:
This should still be able to be minified. Something else must be wrong if you're getting a 404 while trying to retrieve this. Here's what I could potentially see could be the problem
```suggestion
    <script src="/webjars/jquery/1.9.1/jquery.min.js"></script>
```

Reply:
Suggested fix doesn't work, but I don't think it can be minified either way because its the maven blob. This is reverting a change from #102

Reply:
The fact it's a maven blob is not the issue, I'm able to use it perfectly fine in the uploader page. That 404 didn't occur when I pulled it. This is likely something else.

Reply:
Well, i have no idea then. When I changed it to what you suggested I got the same error, but when I change it to the non-minified version I don't get the error.

Reply:
I tried pulling up the editor page on what's currently deployed to heroku and I'm not seeing the 404 in the console. Maybe just something wrong on your end?

Reply:
![image](https://user-images.githubusercontent.com/10526581/54566809-4c99ec00-49a8-11e9-9a5a-e4ddcfac10b2.png)

Jquery is importing perfectly fine.

Reply:
Ok I will close this issue